zebra/label_manager.c

```diff
@@ -138,38 +149,150 @@ void lm_hooks_register(void)
 	hook_register(lm_client_disconnect, label_manager_disconnect);
 	hook_register(lm_get_chunk, label_manager_get_chunk);
 	hook_register(lm_release_chunk, label_manager_release_label_chunk);
+	hook_register(lm_write_label_block_config,
+		      label_manager_write_label_block_config);
 }
 void lm_hooks_unregister(void)
 {
 	hook_unregister(lm_client_connect, label_manager_connect);
 	hook_unregister(lm_client_disconnect, label_manager_disconnect);
 	hook_unregister(lm_get_chunk, label_manager_get_chunk);
 	hook_unregister(lm_release_chunk, label_manager_release_label_chunk);
+	hook_unregister(lm_write_label_block_config,
+			label_manager_write_label_block_config);
 }
 
-DEFPY(show_label_table, show_label_table_cmd, "show debugging label-table",
+static json_object *lmc_json(struct label_manager_chunk *lmc)
+{
+	json_object *json = json_object_new_object();
+
+	json_object_string_add(json, "protocol", zebra_route_string(lmc->proto));
+	json_object_int_add(json, "instance", lmc->instance);
+	json_object_int_add(json, "session-id", lmc->session_id);
+	json_object_int_add(json, "start", lmc->start);
+	json_object_int_add(json, "end", lmc->end);
+	if (lmc->is_dynamic)
+		json_object_boolean_true_add(json, "dynamic");
```

Comment:
Just `json_object_boolean()` instead of json_object_boolean_true_add/json_object_boolean_false_add

Reply:
done